### PR TITLE
added missing value prop and added clear input after onclick submit

### LINKED ISSUE #697: SEO redirect creation form isn't reset after redirects are created.
**Describe the bug**
After a redirect is created the form does not reset. It becomes a confusing experience  because it looks like the redirect wasn't created, when in fact it was. 

**To Reproduce**
1. Go to SEO
2. Enter a relative path in the From field. And select a target from the To dropdown.
3. Click Create Redirect.
4. See that even though the redirect is created the form does not reset. 

**Expected behavior**
When redirects are successfully created the form should reset (previous values should be removed).

**Screenshots**

Screen Shot 2021-05-11 at 3 02 05 PM

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser Chrome
 - Version 89??

**Additional context**

### PR DESCRIPTION
**Pull request by d88naimi**
Input fields now  reset after submission

fixes #697

### CODE REVIEW

File: src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js [hunk 1]
```diff
@@ -26,6 +26,8 @@ export function RedirectCreator(props) {
         code: code === 1 ? 301 : 302 // API expects a 301/302 value
       })
     );
```
Comment (shrunyan) [inline src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js:28]: Assuming the `createRedirect` is returning the fetch promise this should be a thenable value. Lets put the reset into a `then` clause. As well as lets ensure a system notification is being shown on when a bad request or error occurs. This way on a bad request we don't wipe the fields and a user can make a correction to try again.

File: src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js [hunk 2]
```diff
@@ -36,6 +38,7 @@ export function RedirectCreator(props) {
           name="redirectFrom"
           type="text"
           placeholder="URL path to redirect from"
+          value={from}
```
Comment (shrunyan) [inline src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js:41]: If this wasn't present before how was the value showing in the field? What does adding this to the field provide us?

**Review by shrunyan — CHANGES_REQUESTED** [review]

File: src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js (hunk 1 above)
Comment (d88naimi) [reply src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js]: add then(), added notify success to the store for successful redirects.

File: src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js (hunk 2 above)
Comment (d88naimi) [reply src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js:43]: I believe the input field was listening on line 45 `evt.target.value` but the input field does not clear after the submit if I dont have line 41 inserted `value={from}`. Please let me know if I am going about this incorrectly thanks.

Comment (shrunyan) [reply src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js:43]: Is there a cursor jump bug now that this is a controlled component?

File: src/apps/seo/src/store/redirects.js [hunk 3]
```diff
@@ -68,13 +68,20 @@ export function createRedirect(redirect) {
...
       method: "POST",
       json: "true",
       body: redirect
     })
       .then(json => {
         if (!json.error) {
+          dispatch(
+            notify({
+              kind: "success",
+              message: `Created new redirect`
```
Comment (shrunyan) [inline src/apps/seo/src/store/redirects.js:81]: We should drop "new" as it's superfluous. "Created" implies it's new.

Comment (shrunyan) [reply src/apps/seo/src/store/redirects.js:81]: For the errors later we should match the creation structure. e.g.
Change: "Failed to create SEO redirect"
To: "Failed creating redirect"

**Review by shrunyan — CHANGES_REQUESTED** [review]

File: src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js (hunk 2 above)
Comment (d88naimi) [reply src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js:43]: No input is cleared, here is a quick video for both working and non working links
https://www.loom.com/share/6e3cd2fdf67242bfbde8ac34d36e7249

Comment (d88naimi) [reply src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js:43]: @shrunyan there is not a cursor jump on the `From` section but there is in `To` section which is happening currently on master prior to my fixes.

Comment (shrunyan) [reply src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js:43]: Ok. Would you split that bug of into a new ticket?

**Pull request merged**